utils/generate-commands-json.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+# Generate a commands.json file from the output of the `COMMAND` command
+# Prerequisites: Python 3.5+ and redis-py
+import argparse
+import functools
+import json
+import redis
+
+def convert_list_to_dict(arr, recurse):
+    rep = {}
+    for i in range(0,len(arr),2):
+        key = arr[i].replace('-', '_')
```

Comment:
I suppose we can drop this now? 
https://github.com/redis/redis/pull/9959

Reply:
Yep